paymentapi/src/main/java/fi/hel/verkkokauppa/payment/paytrail/context/PaytrailPaymentContext.java

```diff
@@ -22,4 +22,17 @@ public class PaytrailPaymentContext {
 
     /* Paytrail Shop-in-Shop merchant ID */
     private String shopId;
+
+    private String cardRedirectSuccessUrl;
+    private String cardRedirectCancelUrl;
+    private String cardCallbackSuccessUrl;
+    private String cardCallbackCancelUrl;
+    private String updateCardRedirectSuccessUrl;
+    private String updateCardRedirectCancelUrl;
+    private String updateCardCallbackSuccessUrl;
+    private String updateCardCallbackCancelUrl;
+    private String redirectSuccessUrl;
+    private String redirectCancelUrl;
+    private String callbackSuccessUrl;
```

Comment:
Could you add some kinf of documentation to these values which does not have prefix ? Eq this is payment callback success url etc

Reply:
Or even better if you could rename these variables to have the prefix